### ISSUE TITLE
python 2 tests are failing

### ISSUE THREAD

**Issue by bannanc** [openforcefield/openforcefield#98]
Do we know why the tests are failing for python2? @jchodera @davidlmobley 
Are you going to continue to support 2?

**Comment by jchodera** [comment]
The python 2 build is failing because it can't find an appropriate OpenEye package:
```
1.73s$ pip install --pre -i https://pypi.anaconda.org/openeye/label/beta/simple openeye-toolkits && python -c "import openeye; print(openeye.__version__)"
Collecting openeye-toolkits
  Downloading https://pypi.anaconda.org/openeye/label/beta/simple/openeye-toolkits/2018.2b2/OpenEye-toolkits-2018.2b2.tar.gz
Collecting OpenEye-toolkits-python2.7-ucs4-linux-x64==2018.2b2 (from openeye-toolkits)
  Could not find a version that satisfies the requirement OpenEye-toolkits-python2.7-ucs4-linux-x64==2018.2b2 (from openeye-toolkits) (from versions: )
No matching distribution found for OpenEye-toolkits-python2.7-ucs4-linux-x64==2018.2b2 (from openeye-toolkits)
```
I think we're going to ditch python 2 support, so we can just disable those tests.

**Comment by jchodera** [comment]
Also, we may want to switch away from the OpenEye `beta` channel:
```
https://pypi.anaconda.org/openeye/label/beta/simple
```
toward the stable channel.

**Comment by davidlmobley** [comment]
Agreed, yes.

**Comment by bannanc** [comment]
I think we have an issue on smarty to move away from beta on OpenEye, I'm working on moving those to the appropriate repository.

**Comment by davidlmobley** [comment]
Resolved by #102
